packages/bpk-component-modal/src/bpk-modal-dialog.scss

```diff
@@ -98,9 +98,11 @@
 
   &__dialog-heading {
     margin: 0;
-    font-weight: 700;
 
     @include bpk-text-base;
```

Comment:
Ok, how should I solve this from a technical aspekt @matthewdavidson?

I'll have to create a token which is the font weight that this mixin sets. In the base theme it will be 700. In the iOS theme, can I make it "false" and check in the mixin if it's a number? I dislike this approach again because it's so specific to the mixin where it's being used.